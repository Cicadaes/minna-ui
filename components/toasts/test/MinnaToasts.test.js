'use strict';

const MinnaToasts = require('../src/MinnaToasts.html');

describe('MinnaToasts component', () => {
  it('should render correctly', () => {
    const target = document.createElement('div');
    new MinnaToasts({ target });
    expect(target.innerHTML).toMatchSnapshot();
  });
});
