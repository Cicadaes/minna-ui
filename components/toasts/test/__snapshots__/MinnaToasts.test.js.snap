// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MinnaToasts component should render correctly 1`] = `""`;
